.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -395,15 +414,16 @@ jobs:
         if: (!endsWith(matrix.make_options, '_INTERP_BUILD_OPTIONS'))
         run: |
           mkdir build && cd build
-          cmake ..
+          cmake -DCMAKE_C_FLAGS="${{ env.SANITIZER_FLAGS }}" -DCMAKE_CXX_FLAGS="${{ env.SANITIZER_FLAGS }}" ..
```

Comment:
I prefer not to set `CMAKE_C_FLAGS` and `CMAKE_CXX_FLAGS` directly. It might hurt a lot. potentially.

Reply:
and what would you do instead?
Unfortunately, this is the only way to enable sanitizers as I know

Reply:
I don't stand for setting `CMAKE_C_FLAGS` and `CMAKE_CXX_FLAGS` from command line directly. I prefer to add such sanitizers into the CMakeLists.txt.

https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/product-mini/platforms/linux/CMakeLists.txt#L133-L157 may not be a perfect but workable example. Or do better to follow "target-based build" rules.

Reply:
Probably having a CLI flag (something like `-DSANITIZER=(asan|tsan|ubsan)`) would work here (and make updates in cmake files as suggested by @lum1n0us )